docs/airgap-install.md

```diff
@@ -57,34 +57,45 @@ Copy the `bundle_file` you created in the previous step or downloaded from the [
 
 As an alternative to the previous step, you can use k0sctl to upload the bundle file to the worker nodes. k0sctl can also be used to upload k0s binary file to all nodes. Take a look at this example (k0sctl.yaml) with one controller and one worker node to upload the bundle file and k0s binary:
 
-```YAML
+```yaml
 apiVersion: k0sctl.k0sproject.io/v1beta1
 kind: ClusterConfig
 metadata:
   name: k0s-cluster
 spec:
-  hosts:
-  - ssh:
-      address: <ip-address-controller>
-      user: ubuntu
-      keyPath: /path/.ssh/id_rsa
-    role: controller
-    uploadBinary: true
-    k0sBinaryPath: /path/to/k0s_binary/k0s
-  - ssh:
-      address: <ip-address-worker>
-      user: ubuntu
-      keyPath: /path/.ssh/id_rsa
-    role: worker
-    uploadBinary: true
-    k0sBinaryPath: /path/to/k0s_binary/k0s
-    files:
-      - name: bundle-file
```

Comment:
The name param is only used to display a custom name in the terminal output, it's a bit useless.